media_types/content_type_test.ts

```diff
@@ -33,13 +33,9 @@ Deno.test({
     _str = contentType("text/html");
     _str = contentType("txt");
 
-    // @ts-expect-error: string | undefined is not assignable to string
```

Comment:
These type errors are expected. See #2348 and #2357 for details

Reply:
Not anymore. Having them there now produces a `Unused '@ts-expect-error' directive.` error.

Reply:
Currently `contentType` returns `string` type if the input is well known type, and it returns `string | undefined` if the input is not well known type (`contentType("foo")` actually returns `undefined`. No type error here means it's typed incorrectly).

These changes of the type expectations means this function loses that feature.

Reply:
Type inference seems to have broken because we're using a `.json` file instead of a `.ts` file. I'll fix it.